filters/shedder/admission.go

```diff
@@ -450,3 +453,10 @@ func (ac *admissionControl) startSpan(ctx context.Context) (span opentracing.Spa
 	}
 	return
 }
+
+// Opt-In for filters to get called Response(ctx) in case of errors
+type errorHandlerFilter interface {
+	HandleErrorResponse()
+}
```

Comment:
I think we do not need it here, its enough to define method (maybe with a Opt-in comment)

Reply:
yeah as staticcheck wrote :)